docs/howto/manage-charm-design.md

```diff
@@ -0,0 +1,65 @@
+(how-to-manage-charm-design)=
+# How to manage charm design
+
+## Module layout
+
+In your `src` folder, use the `charm.py` file provided by `charmcraft init` to
+define the charm class and manage the charm's interface with Juju: set up the
+event observation and add the handlers. For each workload that the charm
+manages, add a `{workload}.py` file that contains methods for interacting with
```

Comment:
Here and everywhere else:

```suggestion
manages, add a `<workload>.py` file that contains methods for interacting with
```

This has been the convention I've followed for placeholder thingies everywhere else.